package.json

```diff
@@ -17,7 +17,7 @@
         "predeploy": "npm run build",
         "deploy": "gh-pages -d build",
         "format": "prettier --write src pages",
```

Comment:
Could you remove the `pages` directory? It's a left over artifact when I was trying out NextJS; we only need to format source files in `src`

Reply:
Done.